lib/AFS/CellCC/Util.pm

```diff
@@ -277,23 +277,26 @@ _unpretty_bytes($) {
 }
 
 # When there is not enough space free to store a dump blob on the local scratch
-# disk, call this to revert the job to the given state.
+# disk, call this to revert the jobs to the given state.
 sub
-_scratch_rollback($$) {
-    my ($job, $state) = @_;
-    update_job(jobid => $job->{jobid},
-               dvref => \$job->{dv},
-               to_state => $state,
-               timeout => undef,
-               description => "Waiting for enough scratch disk space to be free");
+_scratch_rollback($@) {
```

Comment:
in general, I'd rather these functions keep the order of their arguments the same; just replace `$job` with an arrayref of jobs. I just think it makes it easier to update the arguments in the future (obv we can only have one array arg, but multiple arrayref/hashref args)